src/common/components/sidebar/annotations-view.js

```diff
@@ -175,7 +175,11 @@ const AnnotationsView = memo(React.forwardRef((props, ref) => {
 		props.onUpdateAnnotations([annotation]);
 	}, []);
 
-	function handlePointerDown() {
+	function handlePointerDown(event) {
+		// Clicking on the rendered content when a contextmenu is open will
+		// lead to pointerup event not firing and the annotation becoming not-selectable
+		// via keyboard until it is clicked.
+		if (event.target.classList.contains("context-menu-overlay")) return;
```

Comment:
I'm trying to reproduce the mentioned issue. Could you help me with the steps to reproduce it?

Reply:
Yeah, it actually is a bit harder to reproduce now than [before](https://github.com/zotero/zotero/issues/4681#issuecomment-2349860842). The recent fix to context menu focus regression must have helped. 

I did get to reproduce it with an unloaded reader tab. After it loads, click on the annotation `...` to open the context menu, then click on the rendered PDF content, and try to navigate to the sidebar with tab. It gets skipped for me. Clicking around resolves the issue, and on subsequent attempts, it doesn't seem to happen again. 

https://github.com/user-attachments/assets/a738587a-5813-42ac-b1f1-4f717d05ca5d

Reply:
It should be fixed in 97dd5aba93d99807a5393ebf9377689e0fc5f803. Thanks for helping to reproduce it.

Reply:
Yeah, it's fixed now. That was definitely a less-roundabout way to do it!

I removed this changed from here.